reporting/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "0.1"
```

Comment:
I'm curious; is there a reason to use version 0.1 here?

Reply:
not really, seemed like a reasonable number to start w/, but I'm not familiar w/what the best practice is and happy w/whatever that is

Reply:
Ah okay, docker-compose files start at version 1, and all our compose files are at least version 2. Probably best to match the main file in the Ref Distro, 3.3.

https://docs.docker.com/compose/compose-file/compose-versioning/